xml/dm_registers.xml

```diff
@@ -650,7 +664,8 @@
         \begin{steps}{Writes to this register start the following:}
             \item Set \Fsbbusy.
             \item Perform a bus write of the new value of {\tt sbdata} to {\tt sbaddress}.
-            \item If \Fsbautoincrement is set, increment {\tt sbaddress}.
+            \item If the write succeeded and \Fsbautoincrement is set,
+            increment {\tt sbaddress}.
             \item Clear \Fsbbusy.
```

Comment:
you actually only set/clear busy if it wasn't already busy, right? This is a little unclear from this formation of the description.

Reply:
I guess the sentence above sort of goes with this. Maybe you should say, "If the bus master is busy then accesses set \Fsberror. Otherwise, {steps}"